docs/tutorials/otio-file-format-specification.md

```diff
@@ -2,9 +2,7 @@
 
 ## Version
 
-    This may be out of date. 
-
-This DRAFT describes the OpenTimelineIO JSON File Format as of OTIO Alpha 5.
+This DRAFT describes the OpenTimelineIO JSON File Format as of OTIO Beta 13.
```

Comment:
Is there a way to embed this into the automatically generated document?  Otherwise we should add to our release process updating this document.

Reply:
Given that we didn't actually update the contents of this since Alpha 5, I think it was correct. We should add a manual process to any schema enhancements to include documentation.

Reply:
Sorry, to include updating the documentation?  Or to include documentation in general?  

Either way, we should keep a list of which pages to check then.  We can add a todo to our TSC documentation pass list.

Reply:
I think both. Specifically any PR that changes schema should update this document (in addition to the autogenerated one) - and when we do a release, we should have a reminder "check that the file format spec documentation reflects any schema changes" in case we missed anything important.

Reply:
Makes sense.  Thanks!